### ISSUE TITLE
Invalid detailedAppStatus sometimes received

### ISSUE THREAD

**Issue by tetchel** [eclipse/codewind#1697]
When doing an initial build on a Node project, the initial detailedAppStatus is briefly an error that fails to be translated:
```
[09/01/20 20:56:57 nodeproj] [ERROR] getContainerStatus : 167 | | The command docker exec,cw-nodeproj-07d5ee40-324e-11ea-9797-253d9e5e9218,/scripts/noderun.sh,isActive failed with exit code 1
[09/01/20 20:56:57 nodeproj] [INFO] pingInTransitApplications: Application state for project 07d5ee40-324e-11ea-9797-253d9e5e9218 has changed from starting to stopped
[09/01/20 20:56:57 nodeproj] [INFO] Emitting event
 message: projectStatusChanged
 data: {
  "projectID": "07d5ee40-324e-11ea-9797-253d9e5e9218",
  "appStatus": "stopped",
  "appErrorStatus": "projectStatusController.appStatusContainerStopped",
  "detailedAppStatus": {
    "severity": "ERROR",
    "message": "projectStatusController.appStatusContainerStopped",
    "notify": true
  }
}
[09/01/20 20:57:01 ProjectList.js] [INFO] Project nodeproj started (project ID: 07d5ee40-324e-11ea-9797-253d9e5e9218)
[09/01/20 20:57:01 nodeproj] [INFO] pingApplications: Application state for project 07d5ee40-324e-11ea-9797-253d9e5e9218 has changed from stopped to started
[09/01/20 20:57:01 nodeproj] [INFO] Emitting event
 message: projectStatusChanged
 data: {
  "projectID": "07d5ee40-324e-11ea-9797-253d9e5e9218",
  "appStatus": "started",
  "detailedAppStatus": {
    "severity": "INFO",
    "message": "",
    "notify": false
  }
```

since `notify: true`, this results in VS Code showing the following: 
![image](https://user-images.githubusercontent.com/8888438/72104196-dda3d000-32f8-11ea-8c57-4857cbd25da1.png)

**Comment by tetchel** [comment]
it looks like this is what's causing the `exit status 1`

```
[09/01/20 20:56:38 nodeproj] [ERROR] executeBuildScript : 383 | + set +x
+ npm run start

```

**Comment by rajivnathan** [comment]
@tetchel can you please verify

**Comment by tetchel** [comment]
Message is fixed as per Erin in #1709
